### PR TITLE
apache-hadoop.rst: Editorial changes

### PR DESCRIPTION
**Pull request by DougTW**
Editorial changes include:
- added standardized introduction paragraph and description
- rearranged some content into introduction
- added contents directive at the start of the document
- revised some steps to be continuations of other steps 

Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: source/tutorials/apache-hadoop.rst [hunk 1]
```diff
@@ -3,33 +3,41 @@
...
+The Apache Hadoop software library is a framework for distributed processing
+of large data sets across clusters of computers using simple programming
+models. It is designed to scale up from single servers to thousands of
+machines, with each machine offering local computation and storage.
 
 Prerequisites
 *************
 
-Before following this tutorial, you should follow the
+Before following this tutorial, follow the
```
Comment (mvincerx) [inline source/tutorials/apache-hadoop.rst]: *  :ref:`bare-metal-install-desktop`  
* In |CL| run `:command: swupd update`

File: source/tutorials/apache-hadoop.rst [hunk 2]
```diff
@@ -3,33 +3,41 @@
...
 
-Before you install any new packages, update |CL| with the following command:
+Before installing any new packages, update |CL| with the following command:
 
 .. code-block:: bash
 
    sudo swupd update
 
-For the purposes of this tutorial, we will install Hadoop in a single machine
+For this tutorial, you will install Hadoop in a single machine
```
Comment (mvincerx) [inline source/tutorials/apache-hadoop.rst]: Please move this sentence to the first in the para under "Description."

File: source/tutorials/apache-hadoop.rst (hunk 1 above)
Comment (DougTW) [reply source/tutorials/apache-hadoop.rst]: Done.

File: source/tutorials/apache-hadoop.rst (hunk 2 above)
Comment (DougTW) [reply source/tutorials/apache-hadoop.rst]: Done.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**